test/TestHelper.cpp

```diff
@@ -172,6 +173,61 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
+bool ImportTest::compareStates(State const& _stateExpect, State const& _statePost)
```

Comment:
Everything you do in this function, I do here: https://github.com/ethereum/cpp-ethereum/blob/develop/test/state.cpp#L99-115
Please use the same helper function and unify this code. So we have no code duplication and can have one simple function for both cases.